bugsnag_performance.go

```diff
@@ -89,7 +89,16 @@ func createBugsnagMergedResource() *resource.Resource {
 		{
 			Key:   "service.version",
 			Value: attribute.StringValue(Config.AppVersion),
-		}}
+		},
+		{
+			Key:   "bugsnag.telemetry.sdk.name",
+			Value: attribute.StringValue("Go Bugsnag Performance SDK"),
```

Comment:
Can we abstract this to make sure it's the same as the User-Agent value and can't drift?

Reply:
Ok, will do, same for Ruby then.